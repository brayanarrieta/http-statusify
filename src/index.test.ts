import * as library from '.';

describe('http-statusify API', () => {
  it('should export the correct structure', () => {
    expect(library).toMatchSnapshot();
  });
});
