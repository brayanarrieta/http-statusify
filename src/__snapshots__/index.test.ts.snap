// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`http-statusify API should export the correct structure 1`] = `
Object {
  "StatusCodes": Object {
    "100": "CONTINUE",
    "101": "SWITCHING_PROTOCOLS",
    "102": "PROCESSING",
    "103": "EARLY_HINTS",
    "200": "OK",
    "201": "CREATED",
    "202": "ACCEPTED",
    "203": "NON_AUTHORITATIVE_INFORMATION",
    "204": "NO_CONTENT",
    "205": "RESET_CONTENT",
    "206": "PARTIAL_CONTENT",
    "207": "MULTI_STATUS",
    "208": "ALREADY_REPORTED",
    "226": "IM_USED",
    "300": "MULTIPLE_CHOICES",
    "301": "MOVED_PERMANENTLY",
    "302": "FOUND",
    "303": "SEE_OTHER",
    "304": "NOT_MODIFIED",
    "305": "USE_PROXY",
    "307": "TEMPORARY_REDIRECT",
    "308": "PERMANENT_REDIRECT",
    "400": "BAD_REQUEST",
    "401": "UNAUTHORIZED",
    "402": "PAYMENT_REQUIRED",
    "403": "FORBIDDEN",
    "404": "NOT_FOUND",
    "405": "METHOD_NOT_ALLOWED",
    "406": "NOT_ACCEPTABLE",
    "407": "PROXY_AUTHENTICATION_REQUIRED",
    "408": "REQUEST_TIMEOUT",
    "409": "CONFLICT",
    "410": "GONE",
    "411": "LENGTH_REQUIRED",
    "412": "PRECONDITION_FAILED",
    "413": "PAYLOAD_TOO_LARGE",
    "414": "URI_TOO_LONG",
    "415": "UNSUPPORTED_MEDIA_TYPE",
    "416": "RANGE_NOT_SATISFIABLE",
    "417": "EXPECTATION_FAILED",
    "418": "IM_A_TEAPOT",
    "421": "MISDIRECTED_REQUEST",
    "422": "UNPROCESSABLE_ENTITY",
    "423": "LOCKED",
    "424": "FAILED_DEPENDENCY",
    "425": "TOO_EARLY",
    "426": "UPGRADE_REQUIRED",
    "428": "PRECONDITION_REQUIRED",
    "429": "TOO_MANY_REQUESTS",
    "431": "REQUEST_HEADER_FIELDS_TOO_LARGE",
    "451": "UNAVAILABLE_FOR_LEGAL_REASONS",
    "499": "CLIENT_CLOSED_REQUEST",
    "500": "INTERNAL_SERVER_ERROR",
    "501": "NOT_IMPLEMENTED",
    "502": "BAD_GATEWAY",
    "503": "SERVICE_UNAVAILABLE",
    "504": "GATEWAY_TIMEOUT",
    "505": "HTTP_VERSION_NOT_SUPPORTED",
    "506": "VARIANT_ALSO_NEGOTIATES",
    "507": "INSUFFICIENT_STORAGE",
    "508": "LOOP_DETECTED",
    "509": "BANDWIDTH_LIMIT_EXCEEDED",
    "510": "NOT_EXTENDED",
    "511": "NETWORK_AUTHENTICATION_REQUIRED",
    "599": "NETWORK_CONNECT_TIMEOUT_ERROR",
    "ACCEPTED": 202,
    "ALREADY_REPORTED": 208,
    "BAD_GATEWAY": 502,
    "BAD_REQUEST": 400,
    "BANDWIDTH_LIMIT_EXCEEDED": 509,
    "CLIENT_CLOSED_REQUEST": 499,
    "CONFLICT": 409,
    "CONTINUE": 100,
    "CREATED": 201,
    "EARLY_HINTS": 103,
    "EXPECTATION_FAILED": 417,
    "FAILED_DEPENDENCY": 424,
    "FORBIDDEN": 403,
    "FOUND": 302,
    "GATEWAY_TIMEOUT": 504,
    "GONE": 410,
    "HTTP_VERSION_NOT_SUPPORTED": 505,
    "IM_A_TEAPOT": 418,
    "IM_USED": 226,
    "INSUFFICIENT_STORAGE": 507,
    "INTERNAL_SERVER_ERROR": 500,
    "LENGTH_REQUIRED": 411,
    "LOCKED": 423,
    "LOOP_DETECTED": 508,
    "METHOD_NOT_ALLOWED": 405,
    "MISDIRECTED_REQUEST": 421,
    "MOVED_PERMANENTLY": 301,
    "MULTIPLE_CHOICES": 300,
    "MULTI_STATUS": 207,
    "NETWORK_AUTHENTICATION_REQUIRED": 511,
    "NETWORK_CONNECT_TIMEOUT_ERROR": 599,
    "NON_AUTHORITATIVE_INFORMATION": 203,
    "NOT_ACCEPTABLE": 406,
    "NOT_EXTENDED": 510,
    "NOT_FOUND": 404,
    "NOT_IMPLEMENTED": 501,
    "NOT_MODIFIED": 304,
    "NO_CONTENT": 204,
    "OK": 200,
    "PARTIAL_CONTENT": 206,
    "PAYLOAD_TOO_LARGE": 413,
    "PAYMENT_REQUIRED": 402,
    "PERMANENT_REDIRECT": 308,
    "PRECONDITION_FAILED": 412,
    "PRECONDITION_REQUIRED": 428,
    "PROCESSING": 102,
    "PROXY_AUTHENTICATION_REQUIRED": 407,
    "RANGE_NOT_SATISFIABLE": 416,
    "REQUEST_HEADER_FIELDS_TOO_LARGE": 431,
    "REQUEST_TIMEOUT": 408,
    "RESET_CONTENT": 205,
    "SEE_OTHER": 303,
    "SERVICE_UNAVAILABLE": 503,
    "SWITCHING_PROTOCOLS": 101,
    "TEMPORARY_REDIRECT": 307,
    "TOO_EARLY": 425,
    "TOO_MANY_REQUESTS": 429,
    "UNAUTHORIZED": 401,
    "UNAVAILABLE_FOR_LEGAL_REASONS": 451,
    "UNPROCESSABLE_ENTITY": 422,
    "UNSUPPORTED_MEDIA_TYPE": 415,
    "UPGRADE_REQUIRED": 426,
    "URI_TOO_LONG": 414,
    "USE_PROXY": 305,
    "VARIANT_ALSO_NEGOTIATES": 506,
  },
}
`;
